src/libcore/num/dec2flt/rawfp.rs

```diff
@@ -56,22 +56,28 @@ impl Unpacked {
 /// Should **never ever** be implemented for other types or be used outside the dec2flt module.
 /// Inherits from `Float` because there is some overlap, but all the reused methods are trivial.
```

Comment:
same as before "inherits from `Float`" is no longer true

Reply:
I’ve now removed this sentence.